includes/admin/class-amp-customizer.php

```diff
@@ -127,41 +71,43 @@ public function register_settings() {
 		do_action( 'amp_customizer_register_settings', $this->wp_customize );
 	}
 
-	public function add_customizer_scripts() {
-		wp_enqueue_script( 'wp-util' ); // fix `wp.template is not a function`
-		do_action( 'amp_customizer_enqueue_scripts' );
-	}
-
 	/**
-	 * Enqueues scripts and fires the 'wp_footer' action so we can output customizer scripts.
-	 *
-	 * This breaks AMP validation in the customizer but is necessary for the live preview.
+	 * Load up AMP scripts needed for Customizer integrations.
 	 *
-	 * @since 0.4
+	 * @since 0.6
 	 * @access public
 	 */
-	public function add_preview_scripts() {
+	public function add_customizer_scripts() {
 		wp_enqueue_script(
 			'amp-customizer',
-			amp_get_asset_url( 'js/amp-customizer-preview.js' ),
-			array( 'jquery', 'customize-preview', 'wp-util' ),
+			amp_get_asset_url( 'js/amp-customize-controls.js' ),
+			array( 'jquery', 'customize-controls' ),
 			$version = false,
-			$footer = true
+			$footer  = true
 		);
 
-		do_action( 'amp_customizer_enqueue_preview_scripts', $this->wp_customize );
+		wp_localize_script( 'amp-customizer', 'ampPost', amp_admin_get_preview_permalink() );
```

Comment:
I'm seeing this output a variable containing `http://src.wordpress-develop.dev/?preview=true&p=53/amp/`. It seems `amp_admin_get_preview_permalink()` has a bug with how it adds the endpoint.

Reply:
Fixed in 0885b48